inventory/rh-sso/group_vars/all.yml

```diff
@@ -14,3 +14,13 @@ rhsm_repos:
 rhsm_username: "{{ lookup('env', 'RHSM_USER' )}}"
 rhsm_password: "{{ lookup('env', 'RHSM_PASSWD' )}}"
 rhsm_pool: "{{ lookup('env', 'RHSM_POOL' )}}"
+
+# User management variables
```

Comment:
This should be called out in a `README` and we don't want to hide this just in the comment. Also as mentioned in other comments, we should look to use a filter to make sure that the username gets set in lowercase.